docs/releases/1.9.txt

```diff
@@ -326,6 +326,13 @@ Generic Views
 * Class based views generated using ``as_view()`` now have ``view_class``
   and ``view_initkwargs`` attributes.
 
+* :func:`~django.utils.decorators.method_decorator` can now be used to decorate
```

Comment:
You could put the link from the last sentence on "decorate classes instead of methods" and omit the last sentence.